### PR TITLE
Remove Undelete

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This is to remove the undelete functionality for the app as we will no longer support undeleting as we will now be implementing a versioning system instead. This is a replacement for a previous PR that included questionnaire deletion confirmation but that is now in a separate PR

### How to review 
- Check the app works as it should
- Check you can delete all types of things (questionnaire, section, question, answer and section introduction)
- Check that when deleting these things the toast is still shown but now there is no undo button

### CODE REVIEW

**Comment by rmccar** [comment]
[Strip undelete functionality.](https://trello.com/c/EEZFnuBX/837-strip-undelete-functionality)

File: eq-author/cypress/integration/authenticated/builder_spec.js [hunk 1]
```diff
@@ -264,42 +264,6 @@ describe("builder", () => {
     cy.get(testId("btn-add-intro"));
   });
 
-  it("can undelete a section introduction", () => {
```
Comment (tgandrews) [inline eq-author/cypress/integration/authenticated/builder_spec.js]: I don't think this is the only undelete cypress test

Comment (rmccar) [reply eq-author/cypress/integration/authenticated/builder_spec.js]: Ive done a search for undelete and cant seem to find any others

Comment (rmccar) [reply eq-author/cypress/integration/authenticated/builder_spec.js]: You're right Ive found some that don't use the word 'undelete'

File: eq-author/src/components/ToastContainer/index.js [hunk 2]
```diff
@@ -55,36 +38,15 @@ const ToastWrapper = styled.div`
   justify-content: center;
 `;
 
-export const ToastArea = ({
-  toasts,
-  dismissToast,
-  undoToast,
-  ...otherProps
-}) => {
```
Comment (SamGodwin2) [inline eq-author/src/components/ToastContainer/index.js]: Don't think you've updated the tests for this component probably a couple broken snapshot tests in there somewhere.

Comment (rmccar) [reply eq-author/src/components/ToastContainer/index.js]: Fixed

**Pull request merged**